File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

void printspaces(int i);
void printhashes(int i);

int main(void)
{
    
    int height;
    do
    {
      printf("Height: ");
      height = GetInt();
    }
    while (height < 0 || height > 23);
    
    for (int i = 1; i <= height; i++)
    {
        int spaces = height - i;
        
        printspaces(spaces);
        printhashes(i);
        printspaces(2);
        printhashes(i);
        printf("\n");
        
    }
    
}

void printspaces(int i)
{
    int n = 0;
    while (n < i)
    {
      printf(" ");  
      n++;
    }
}

void printhashes(int i)
{
    int n = 0;
    while (n < i)
    {
      printf("#");
      n++;
    }
}

File: pset6/mario.py
import cs50

def main():
    while True:
        print("Height: ", end="")
        height = cs50.get_int()
        if height >= 0 or height < 23:
            break
    
    for i in range(height):
        spaces = height - (i + 1)
        print(" " * spaces, end="")
        print("#" * (i + 1), end="")
        print(" " * 2, end="")
        print("#" * (i + 1), end="")
        print()

if __name__ == "__main__":
    main()